core/src/test/java/apoc/periodic/PeriodicTest.java

```diff
@@ -79,6 +79,18 @@ public void testSubmitStatement() throws Exception {
         testCall(db, callList, (r) -> assertEquals(true, r.get("done")));
     }
 
+    @Test
+    public void testSubmitWithSchemaOperation() {
+        try {
+            testCall(db, "CALL apoc.periodic.submit('subSchema','CREATE INDEX periodicIdx FOR (n:Bar) ON (n.first_name, n.last_name)')",
```

Comment:
When you cherry-pick to 4.x, it would be good to also add a test for `CALL db.createUniquePropertyConstraint` (that has been removed in Neo4j 5.0) to make sure this fix the original reported issue.